src/Deal/Dto/DealHubSpotEntity.cs

```diff
@@ -27,7 +27,7 @@ public DealHubSpotEntity()
         [DataMember(Name = "pipeline")]
         public string Pipeline { get; set; }
         [DataMember(Name = "hubspot_owner_id")]
-        public string OwnerId { get; set; }
+        public long? OwnerId { get; set; }
```

Comment:
**comment** it's fine that we break it, but we need to list it as a breaking change when releasing.